examples/mutual-ssl/mutual_ssl.bal

```diff
@@ -13,46 +14,48 @@ http:ServiceEndpointConfiguration helloWorldEPConfig = {
             path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
             password: "ballerina"
         },
+         // Configure the preferred SSL protocol and the versions to enable.
         protocol: {
             name: "TLS",
             versions: ["TLSv1.2", "TLSv1.1"]
         },
+
+         // Configure the preferred ciphers.
         ciphers: ["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"],
-        // Enable mutual SSL.
+
+         // Enable mutual SSL.
         sslVerifyClient: "require"
+
     }
 };
 
+// Create a listener endpoint.
 listener http:Listener helloWorldEP = new(9095, config = helloWorldEPConfig);
 
 @http:ServiceConfig {
     basePath: "/hello"
 }
 
-// Bind the service to the endpoint that you declared above.
+// Bind the service to the listener endpoint that you declared above.
```

Comment:
```suggestion
// Bind the service to the listener endpoint that you declared earlier.
```

The word "above" doesn't work in this context because comments appear on the side of the code when we render this on ballerina.io.